ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -357,9 +350,7 @@ export const itwEidIssuanceMachine = setup({
               target: "DisplayingPreview"
             },
             {
-              actions: assign(
-                setFailure(IssuanceFailureType.NOT_MATCHING_IDENTITY)
-              ),
+              actions: "setFailure",
```

Comment:
Did we lose the `NOT_MATCHING_IDENTITY` error?

Reply:
Nice catch, thanks. I restored the manual mapping for these cases, we can later review the errors in `io-react-nativ-e-wallet` for a better handling in the app. Let me know what you think.